package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "lint": "semistandard",
     "prepublish": "./downloadCurrentVersion.js && ./verifyVersion.js",
+    "pretest": "npm run lint",
```

Comment:
Didn't know this existed. CI was running it manually before the test.

Reply:
For anyone checking, here's the complete list: https://docs.npmjs.com/misc/scripts